### DISCUSSION TITLE
How to set sorting of playlists to custom order?

### DISCUSSION THREAD

**Discussion by KenthJ** [JohnnyCrazy/SpotifyAPI-NET#959] [Q&A]
In the official Spotify desktop app, it is possible to set the order to my playlists to 'Recents', 'Alphabetical', 'Custom' etc

I always use 'Custom' here, so I can rearrange the order of my playlists as I like.

I have been using this Spotify-Net Api for the last 2 years, are are super happy for it.
The Api usually sorts my playlists in this custom orders, but suddenly something has changed, so it now order it by 'Recents' instead  of 'Custom'
I'm not aware that I actively have changed anything, and I can't find anywhere where I can specify in the API which sort order I want my playlists to be in.

Any help is greatly appreciated

**Comment by JohnnyCrazy** [comment]
Hi,

The [playlists endpoint](https://developer.spotify.com/documentation/web-api/reference/get-a-list-of-current-users-playlists) does not offer a sort parameter, so this is not possible. Only way is to fetch all playlists and then sort in your application manually.
